src/processors/other/Delay.cpp

```diff
@@ -31,12 +45,17 @@ ParamLayout DelayModule::createParameterLayout()
     using namespace ParameterHelpers;
     auto params = createBaseParams();
 
-    createTimeMsParameter (params, "time_ms", "Delay Time", createNormalisableRange (20.0f, 2000.0f, 200.0f), 100.0f);
-
-    createFreqParameter (params, "freq", "Cutoff", 500.0f, 10000.0f, 4000.0f, 10000.0f);
-    createPercentParameter (params, "feedback", "Feedback", 0.0f);
-    createPercentParameter (params, "mix", "Mix", 0.5f);
-
+    createTimeMsParameter (params, delayTimeMsTag, "Delay Time", createNormalisableRange (20.0f, 2000.0f, 200.0f), 100.0f);
+    createFreqParameter (params, freqTag, "Cutoff", 500.0f, 10000.0f, 4000.0f, 10000.0f);
+    createPercentParameter (params, feedBackTag, "Feedback", 0.0f);
+    createPercentParameter (params, mixTag, "Mix", 0.5f);
+
+    emplace_param<AudioParameterChoice> (params,
+                                         tempoSyncAmountTag,
+                                         "Tempo Sync Delay",
```

Comment:
I think this name is a little bit long... since the other parameter is called "Delay Time", maybe this one should be "Delay Rhythm"?